### PR TITLE
Release Notes for 1.17.3 release.

### PR DESCRIPTION
**Pull request by johnma14**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: .spelling [hunk 1]
```diff
@@ -200,6 +200,7 @@ Cale
 CDNs
 CentOS
 Cernich
+cert
```
Comment (ericvn) [inline .spelling:203]: I don't believe cert is a valid word, although it may be an abbreviation. Remove this here, and update cert to certificate below.

Comment (ericvn) [reply .spelling:203]: At the same time we do have cert in other areas so I don't think there was any error saying cert is actually misspelled and you might be able to just leave them.

File: .spelling [hunk 2]
```diff
@@ -519,6 +520,8 @@ intra-cluster
 intrahost
 intranode
 Invasiveness
+iop
+iops
```
Comment (ericvn) [inline .spelling:524]: iop is not a valid word, so these should be removed as well. IOP is an ambiguous abbreviation.

I see the word used in other docs, dos don't necessarily think it was being flagged as an error either.

Comment (ericvn) [reply .spelling:524]: OK. looks like maybe `iops` was flagged. Not sure if IOPs would fair better.

File: content/en/news/releases/1.17.x/announcing-1.17.3/index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+- **Fixed** an issue where `Istio Gateway` (Envoy) would crash due to a duplicate `istio_authn` network filter in the Envoy filter chain. [Issue #44385](https://github.com/istio/istio/issues/44385)
+- **Fixed** the VirtualService validation to fail on empty prefix header matcher. [Issue #44424](https://github.com/istio/istio/issues/44424)
+- **Fixed** a bug where services are missing in gateways if `PILOT_FILTER_GATEWAY_CLUSTER_CONFIG` is enabled. [Issue #44439](https://github.com/istio/istio/issues/44439)
+- **Fixed** `istioctl analyze` no longer expects pods and runtime resources when analyzing files. [Issue #40861](https://github.com/istio/istio/issues/40861)
+- **Fixed** `istioctl verify-install` fails when using multiple iops. [Issue #42964](https://github.com/istio/istio/issues/42964)
+- **Fixed** handling of remote SPIFFE trust bundles containing multiple certs.[PR #44909](https://github.com/istio/istio/pull/44909)
+- **Fixed** cpu usage abnormally high when cert specified by DestinationRule are invalid. [Issue #44986](https://github.com/istio/istio/issues/44986)
```
Comment (ericvn) [inline content/en/news/releases/1.17.x/announcing-1.17.3/index.md:26]:
```suggestion
- **Fixed** CPU usage abnormally high when cert specified by DestinationRule are invalid. [Issue #44986](https://github.com/istio/istio/issues/44986)
```

**Comment by ericvn** [comment]
LGTM. Will likely need to change the dat.

File: content/en/news/releases/1.17.x/announcing-1.17.3/index.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+- **Fixed** the VirtualService validation to fail on empty prefix header matcher. [Issue #44424](https://github.com/istio/istio/issues/44424)
+- **Fixed** a bug where services are missing in gateways if `PILOT_FILTER_GATEWAY_CLUSTER_CONFIG` is enabled. [Issue #44439](https://github.com/istio/istio/issues/44439)
+- **Fixed** `istioctl analyze` no longer expects pods and runtime resources when analyzing files. [Issue #40861](https://github.com/istio/istio/issues/40861)
+- **Fixed** `istioctl verify-install` fails when using multiple IOPs. [Issue #42964](https://github.com/istio/istio/issues/42964)
+- **Fixed** handling of remote SPIFFE trust bundles containing multiple certificates.[PR #44909](https://github.com/istio/istio/pull/44909)
+- **Fixed** CPU usage abnormally high when cert specified by DestinationRule are invalid. [Issue #44986](https://github.com/istio/istio/issues/44986)
+- **Fixed** `ValidatingWebhookConfiguration` not being generated correctly with operator installation when the revision is not set. [Issue #43893](https://github.com/istio/istio/issues/43893)
```
Comment (ericvn) [inline content/en/news/releases/1.17.x/announcing-1.17.3/index.md:27]: And this one should be what was reverted, so remove if we are going with that.

Comment (johnma14) [reply content/en/news/releases/1.17.x/announcing-1.17.3/index.md]: done.

**Comment by johnma14** [comment]
@ericvn I think the release notes are up-to-date with the requested changes. Can you take a look? Thanks.

**Pull request merged**